R/class-auth.R

```diff
@@ -41,11 +41,18 @@ FirebaseAuth <- R6::R6Class(
 #' @details Print the class
     print = function(){
       cli_rule("Firebase Authentication")
-      signin <- "No user is signed in"
-      if(isTRUE(private$.user_signed_in$signed_in))
-        signin <- "A user is signed in"
-      
-      cli_alert_info(signin, "\n")
+      user <- self$get_signed_in()
+      if (!isTRUE(user$success)) {
+        cli_alert_info("No user is signed in")
+      } else {
+        cli_alert_info("A user is signed in")
+        cli::cli_dl(c(
+          uid = user$response$uid,
+          email = user$response$email,
+          emailVerified = user$response$emailVerified,
```

Comment:
This class is used as parent for all other auth methods.
Some of these fields in the `user` object may not be available, e.g. `emailVerified`.
If we want clearer `print` methods we can override in children classes.